jest.config.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   },
   verbose: true,
   clearMocks: true,
-  testTimeout: 180000,
+  testTimeout: 580000,
```

Comment:
yea.. trying to not extend the global test timeout

Reply:
yea gotcha. cant seemed to fix that from my side atm

Reply:
seems to be a jest v27 issue from what i have read. bypass with something like this 

``` javascript
it('',<callback>, <timeout>)
```